zerver/lib/actions.py

```diff
@@ -656,7 +656,7 @@ def do_set_realm_signup_notifications_stream(realm: Realm, stream: Stream,
     )
     send_event(realm, event, active_user_ids(realm.id))
 
-def do_deactivate_realm(realm: Realm) -> None:
+def do_deactivate_realm(realm: Realm, acting_user: Optional[UserProfile]=None) -> None:
```

Comment:
I'll also add a commit that makes the support code path record which user deactivated it.  It's good practice.

We should probably do a more general pass on adding `acting_user` for all the operations in here; could be very helpful if someone breaks into a Zulip server-level admin account to have it be super clear which one and be able to query in the database for all actions they took.  Probably its own issue though.